JavaAssignment/src/rahulM/WeekdayName.java

```diff
@@ -0,0 +1,42 @@
+package switchcase;
+
+import java.util.Scanner;
+
+public class WeekdayName {
+ void printDayName(){
+	 Scanner sc = new Scanner(System.in);
+	 System.out.println("Enter the input number: ");
```

Comment:
Here, not to write scanner class. because if some one want to use your method with his parameter then how it will use your method. so try give parameter to the method and scanner class should be in main method.